workflows/raredisease.nf

```diff
@@ -189,8 +188,7 @@ workflow RAREDISEASE {
                                                                               : Channel.value([])
 
     // Generate pedigree file
-    MAKE_PED (CHECK_INPUT.out.samples.toList())
-    ch_versions = ch_versions.mix(MAKE_PED.out.versions)
+    pedfile = CHECK_INPUT.out.samples.toList().map { make_ped(it) }
```

Comment:
what does the `map` operator do here?

Reply:
It opens the channel and passes the contents of the channel to the function.